File: pricing/list-phone-number-countries/list-phone-number-countries.py
# Download the Python helper library from twilio.com/docs/python/install
from twilio.rest import TwilioPricingClient

# Your Account Sid and Auth Token from twilio.com/user/account
account_sid = "{{ account_sid }}"
auth_token = "{{ auth_token }}"
client = TwilioPricingClient(account_sid, auth_token)

countries = client.phone_numbers.countries.list()

for c in countries:
    print(c.iso_country)

File: pricing/list-phone-number-countries/list-phone-number-countries.js
// Download the Node helper library from twilio.com/docs/node/install
// These vars are your accountSid and authToken from twilio.com/user/account
var accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
var authToken = "your_auth_token";
var PricingClient = require('twilio').PricingClient;
var client = new PricingClient(accountSid, authToken);

client.phoneNumbers.countries.list(function(error, countries) {
    countries['countries'].forEach(function(c){
        console.log(c.isoCountry + "\n");
    });
});